flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/detail/job-checkpoints-detail.component.ts

```diff
@@ -80,5 +97,9 @@ export class JobCheckpointsDetailComponent implements OnInit {
       this.cdr.markForCheck();
       this.refresh();
     });
+    this.jobService.loadCheckpointConfig(this.jobDetail.jid).subscribe(config => {
```

Comment:
these new lines could be removed after refactoring refresh func

Reply:
done